prime-router/settings/staging/0149-etor.yml

```diff
@@ -57,8 +57,8 @@
         numberPerDay: 1440 # Every minute
         initialTime: 00:00
       translation: !<HL7>
-        # THIS WAS schemaName: "metadata/hl7_mapping/STLTs/CO/CO"
-        schemaName: "metadata/hl7_mapping/STLTs/Flexion/TrustedIntermediaryLabOrder"
+        # THIS WAS schemaName: "metadata/hl7_mapping/receivers/STLTs/CO/CO"
+        schemaName: "metadata/hl7_mapping/receivers/Flexion/TrustedIntermediaryLabOrder"
```

Comment:
why is this one `/receivers/` but the other two in this file referring to TrustedIntermediaryLabOrder are both` /STLTs`?

Reply:
I'm not sure I follow, I see that there are a few `STLTs` references, but those are the state specific schema references at this point like line 83 : `schemaName: "metadata/hl7_mapping/receivers/STLTs/CO/CO"`

Reply:
So this line has `metadata/hl7_mapping/receivers/Flexion/TrustedIntermediaryLabOrder` while line 40 has `schemaName: "metadata/fhir_transforms/senders/Flexion/TrustedIntermediaryLabOrder"`....one receivers =, one senders in the path, yet both are for the sane company. Is that expected?

Reply:
Ah gotcha, yea basically it can be organized however, but I thought it would be cleaner to have separate folders for sender/receiver settings. In practice, `hl7_mapping` will only have receiver settings, but since FHIR->FHIR transforms are possible on the sender or receiver side, having the folders reinforce which is which seemed useful.